python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -654,13 +690,36 @@ def log_psf(msg):
         # Final measurement with the CRs removed.
         self.psf_source_measurement.run(detections.sources, exposure)
 
-        # PSF is set on exposure; only return candidates for optional saving.
+        # PSF is set on exposure; candidates are returned to use for
+        # calibration flux normalization and aperture corrections.
         return detections.sources, background, psf_result.cellSet
 
+    def _normalize_calibration_flux(self, exposure, bright_sources):
```

Comment:
I'm not sure this needs to be a separate method from `_measure_aperture_correction`; I think I'd rename the latter to some combination and do both steps in there, since I don't see that we'd need either of them separately and they really do go together.

Reply:
You know, it probably doesn't.  It needed to be if `bright_sources` was an astropy table, but now that I have fixed that up this can be merged again.